h/assets.py

```diff
@@ -2,9 +2,41 @@
 import re
 
 from deform.field import Field
+from webassets.filter import ExternalTool, register_filter
 import pyramid
 
 
+class Browserify(ExternalTool):
+    """
+    An input filter for webassets that browserifies CoffeeScript or JavaScript.
+
+    The browserify command-line client has several limitations when piping
```

Comment:
I think I'd feel comfortable if browserify were the first thing in the chain. My expectation is that most use cases would have browserify as the _only_ tool in the chain and if you needed additional transformation it would be done via browserify plugins/transformations that I would hope we can pass somehow.